examples/step-1/step-1.cc

```diff
@@ -15,24 +15,25 @@
 
  */
 
+// clang-format off
 // @sect3{Include files}
 
 // The most fundamental class in the library is the Triangulation class, which
 // is declared here:
-#include <deal.II/grid/tria.h>
+%:include <deal.II/grid/tria.h>
```

Comment:
It looks like GitHub's syntax highlighting is not standards-compliant :(

Reply:
... and for some reason, `clang-format` messes everything up pretty badly.

Reply:
Well, I vote to remove clang-format and to go back to astyle then.

Reply:
That's a shame. :-(

I thought clang-format would support modern C++ and now it turns out it cannot even deal with digraphs.

Alternatively, I suggest we write a small replacement lexer for clang that we can feed into our own clang-format executable that internally translates between standard syntax and digraphs.

Reply:
Please open a separate pull request for that.

Reply:
on it.